.ci/311-DEV.yaml

```diff
@@ -3,7 +3,10 @@ channels:
   - conda-forge
 dependencies:
   - python=3.11
-  - geopandas>=0.10.0
+  - geopandas
+  - folium
```

Comment:
Do we use folium here? In any case, it will come with geopandas from conda.

Reply:
Ah, forgot to remove that when I added geopandas back into the `conda install` portion of the environment. Will take care of that.